lib/fastly/client.rb

```diff
@@ -57,6 +61,10 @@ def authed?
 
     # Some methods require full username and password rather than just auth token
     def fully_authed?
+     has_both_username_and_password?
+    end
+
+    def has_both_username_and_password?
```

Comment:
Since the two methods do exactly the same thing, I'd prefer to just have one of them. If you prefer the name `has_both_username_and_password?` better than `fully_authed?`, let's go ahead and rename the method—perhaps as a separate pull request. For now, would you mind reverting this change?